src/components/measurements/measurementsD3.js

```diff
@@ -305,7 +305,12 @@ export const drawMeansForColorBy = (ref, svgData, treeStrainColors) => {
     // 2 x subplotPadding for padding around the overall mean display
     const ySpacing = (layout.subplotHeight - 4 * layout.subplotPadding) / (numberOfColorByAttributes - 1);
     let yValue = layout.subplotPadding;
-    Object.entries(colorByGroups).forEach(([attribute, {color, values}]) => {
+    // Order the color groups by the legend value order so that we have a stable order for the means
+    orderBy(
```

Comment:
I'm not against lodash per-se (we use it a little bit in lots of places in auspice, and `orderBy` is used elsewhere in the measurements panel) but each time I feel there's a trade-off between learning another function vs writing out the slightly longer JS code. This trade-off varies across different lodash functions. Here I think a `sort` call would be simpler (at least, for others reading the code), but I'm interested in your general thoughts here? (It's not a request to change, especially as we're already importing it elsewhere).

Reply:
`lodash` is my personal preference because I trust it to do the right thing more than my own version of a `sort` call. I think that has definitely made me over-reliant on `lodash` and then miss the simpler ways of doing things like your suggestion above.